README.md

```diff
@@ -133,7 +133,7 @@ weigh(small_lm)
 #> # A tibble: 25 × 2
 #>    object            size
 #>    <chr>            <dbl>
-#>  1 terms         8.06    
```

Comment:
This is the substantive change, i.e. the current wrong output being shown in our rendered README.

Reply:
Thanks for the callout!